handler.go

```diff
@@ -0,0 +1,108 @@
+package asynqmon
+
+import (
+	"net/http"
+
+	"github.com/go-redis/redis/v8"
+	"github.com/gorilla/mux"
+
+	"github.com/hibiken/asynq"
+)
+
+type HandlerOptions struct {
+	RedisClient          redis.UniversalClient
```

Comment:
I prefer not to expose `redis` package from our APIs. (making unnecessary version update to update redis version or if we decide to switch to other redis client package)
Please change this to 
```
RedisConnOpt asynq.RedisConnOpt
```

and instantiate a `redis.UniversalClient` from the option.